### PR TITLE
fixing the date picker

### PR DESCRIPTION
**Pull request by Ddungyov**

### CODE REVIEW

File: data/tokens.json [hunk 1]
```diff
@@ -1826,19 +1826,19 @@
       "type": "sizing"
     },
     "gtmds_date-picker-option_background-color": {
-      "value": "{gtmds_color_white}",
+      "value": "{gtmds_color_white}01",
```
[the remark this answers is not in the document]
Comment (Ddungyov) [reply data/tokens.json:1829]: оpacity 😆

[the remark this answers is not in the document]
Comment (StefanIvanov) [reply data/tokens.json:1829]: rgba({gtmds_color_white}, 0.1)

[the remark this answers is not in the document]
Comment (Ddungyov) [reply data/tokens.json]: fixed :). Didn't knew that, love it.

File: data/tokens.json [hunk 2]
```diff
@@ -1826,19 +1826,19 @@
       "type": "sizing"
     },
     "gtmds_date-picker-option_background-color": {
-      "value": "{gtmds_color_white}",
+      "value": "({gtmds_color_white}, 0.1)",
```
Comment (StefanIvanov) [inline data/tokens.json:1829]:
```suggestion
      "value": "rgba({gtmds_color_white}, 0.1)", 
```

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Review by StefanIvanov — APPROVED** [review]

**Pull request merged**